build.cake

```diff
@@ -202,28 +201,28 @@ Task("Build")
 
         Information("Publishing .NET Core & Standard projects so that dependencies are present...");
 
-        foreach(var framework in NETSTANDARD_FRAMEWORKS)
+        foreach(var framework in new [] { "netstandard1.6", "netstandard2.0", "netcoreapp3.1" })
              MSBuild(ENGINE_CSPROJ, CreateMSBuildSettings("Publish")
                 .WithProperty("TargetFramework", framework)
                 .WithProperty("PublishDir", BIN_DIR + framework));
 
-        foreach(var framework in NETSTANDARD_FRAMEWORKS)
+        foreach(var framework in new [] { "netstandard1.6", "netstandard2.0" })
              MSBuild(ENGINE_API_CSPROJ, CreateMSBuildSettings("Publish")
                 .WithProperty("TargetFramework", framework)
                 .WithProperty("PublishDir", BIN_DIR + framework));
 
-        foreach(var framework in NETCORE_FRAMEWORKS)
+        foreach(var framework in new [] { "netcoreapp1.1", "netcoreapp2.1", "netcoreapp3.1" })
```

Comment:
Could we give this list of frameworks a descriptive name since it repeats?

Reply:
So the lists of targets for each build are now different. The engine publishes for netstandard1.6, netstandard2.0 and netcoreapp3.1. The API dll for the two netstandard targets, and the tests for three net core targets. 

That was my original reason for getting rid of the variables - and I'm not too sure how we could name the three different lists helpfully - being explicit and inlining the lists was the best I could come up with. Any suggestions?

Reply:
Oh, I misread. This seems fine!